### PR TITLE
added Simulation.percent_complete property and fixed function name

### PR DESCRIPTION
**Pull request by bemcdonnell**
Addresses #88

### CODE REVIEW

File: pyswmm/simulation.py [hunk 1]
```diff
@@ -370,4 +370,29 @@ def current_time(self):
...
+        ...     sim.report()
+        >>>
+        >>> 0.01
+        >>> 0.25
+        >>> 0.50
+        >>> 0.75
+        """
+        dt = self.current_time - self.start_time
+        total_time = self.end_time - self.start_time
+        return dt.total_seconds() / total_time.total_seconds()
```
Comment (goanpeca) [inline pyswmm/simulation.py]: Are these values ints or floats? cause we might get undesired integer division. Maybe

`dt.total_seconds()*1.0 / total_time.total_seconds()`

To guarantee they are always floats? (Py2/Py3 gotchas)

Comment (bemcdonnell) [inline pyswmm/simulation.py]: How about float(dt.total_seconds()) / total_time.total_seconds()?

Comment (goanpeca) [inline pyswmm/simulation.py]: Its the same...

File: pyswmm/swmm5.py [hunk 2]
```diff
@@ -24,6 +22,8 @@
 
 # Local imports
 from pyswmm.lib import DLL_SELECTION
+import ctypes
```
Comment (bemcdonnell) [inline pyswmm/swmm5.py]: @goanpeca, ciocheck seems to be moving this...  Setting issue?

Comment (goanpeca) [inline pyswmm/swmm5.py]: add ctypes to `known_standard_library: ctypes`

Comment (bemcdonnell) [inline pyswmm/swmm5.py]: ok

**Pull request merged**